Makefile

```diff
@@ -4,8 +4,7 @@ dependencies:
 serve: dependencies
 	hugo server \
 		--buildDrafts \
-		--buildFuture \
-		--disableFastRender
+		--buildFuture
```

Comment:
The hugo docs state: `buildFuture` doesn't set which content is scheduled for the future. That's the task of the `publishdate`. When that front matter variable is set to a date in the future, Hugo won't render the content. And when `publishdate` is in the past (relative to the current date), the content gets rendered.

I'm asking just to ensure it is the behaviour we want :)